### PR TITLE
drivers/amlogic: export symbols needed to build dvb driver

### PR DESCRIPTION
**Pull request by afl1**
This PR about exporting symbols needed to build  dvb driver as a loadable kernel modul.

### CODE REVIEW

File: drivers/amlogic/amports/Makefile [hunk 1]
```diff
@@ -124,8 +124,5 @@ obj-y += arch/
 obj-y += cnm/
 obj-y += decoder/
 
-ifneq ($(CONFIG_AM_DVB),)
-##EXTRA_CFLAGS = -DENABLE_DEMUX_DRIVER -DMODIFY_TIMESTAMP_INC_WITH_PLL -DCALC_CACHED_TIME #-DUSE_VMASTER_MODE
+i##EXTRA_CFLAGS = -DENABLE_DEMUX_DRIVER -DMODIFY_TIMESTAMP_INC_WITH_PLL -DCALC_CACHED_TIME #-DUSE_VMASTER_MODE
```
Comment (kszaq) [inline drivers/amlogic/amports/Makefile]: Typo?

**Comment by kszaq** [comment]
Thank you for this PR.

Perhaps I'm nitpicking but how is this different from https://raw.githubusercontent.com/kszaq/LibreELEC.tv/libreelec-8.2/projects/S905/patches/linux/export-symbols-needed-to-build-dvb-driver.patch ?

File: drivers/amlogic/amports/Makefile (hunk 1 above)
Comment (afl1) [reply drivers/amlogic/amports/Makefile]: Yes

**Comment by afl1** [comment]
It's same. I originally introduces this patch and you only used it when you ported my driver to your builds.

**Review by Ray-future — APPROVED** [review]

**Comment by kszaq** [comment]
@afl1 Okay. I saw today's commit date and thought you added something.

Look good to me. :+1:

**Pull request merged**